index.js

```diff
@@ -240,6 +269,14 @@ const legacyScriptingRunner = (Zap, zobj, app) => {
 
     const fullMethod = fullMethodName ? Zap[fullMethodName] : null;
     if (fullMethod) {
+      if (options.resetRequestForFullMethod) {
+        // Used by custom fields
+        _.extend(bundle.request, {
+          method: 'GET',
+          url: ''
+        });
```

Comment:
Tested in the Zap Editor, looks like `bundle.request.url` is emptied when it's passed to a full custom fields scripting method, such as `KEY_custom_action_fields` and `KEY_custom_action_result_fields`. But it's not the case for `KEY_poll` and `KEY_write`.